src/components/Background/Backgound.css

```diff
@@ -0,0 +1,12 @@
+.background-image {
+  z-index: 1;
```

Comment:
`z-index` wont work without a `position` property

Reply:
@znjs thanks for reviewing. I have directly added position utility class from my component library, it's visible in html structure.